source/install/compose_guide.rst

```diff
@@ -58,7 +58,7 @@ clients to allow this particular insecure registry.
 To configure the machine's docker client to allow the internal registry, edit
 the docker engine parameters as follows.::
 
-  echo '{ "insecure-registries":["iotmid-docker:5000"] }' | sudo tee /etc/docker/daemon.json
+  echo '{ "insecure-registries":["iotmid-docker.cpqd.com.br:5000"] }' | sudo tee /etc/docker/daemon.json
```

Comment:
Yes, it would. (actually that would deprecate this entire section)
The reason why I haven't done so yet is because we don't have the components there yet.

I'm thinking we might want to postpone merging this change until we release 0.0.0-beta8, and then update this accordingly.

Would that be ok?